The =~ operator was introduced in Bash 3.

This code path is not executed with --disable-cocoa.

--- configure.ac.orig
+++ configure.ac
@@ -269,7 +269,7 @@
    [ac_cv_my_have_cocoa="yes"])
   CFLAGS="$save_CFLAGS"
   if test "${ac_cv_my_have_cocoa}" = "yes"; then
-    [[[ "$target_os" =~ [0-9]+ ]]] && darwin_ver="${BASH_REMATCH[[0]]}"
+#   [[[ "$target_os" =~ [0-9]+ ]]] && darwin_ver="${BASH_REMATCH[[0]]}"
     case x${target} in
     xpowerpc*darwin*)
       # 10.3 needed to link with X11
